### PR TITLE
Add the abilty to style the tabbed card

### PR DESCRIPTION
**Pull request by jangolano**
## Hey, I just made a Pull Request!

Added the ability to style the tabbedcard component.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by freben — COMMENTED** [review]
You should use the theming capabilities in general, latching onto the `BackstageTabbedCard` name that you see in the `makeStyles`. Having a single `style` prop makes it very unclear what it applies to, in a composed/complex component. Could you find a solution where you instead use the styling system to pass down an adapted style to the card, or use a local theme tweak as part of the react tree or so, where you use the raw Card styling capabilities?

Some of the docs starting here can be of help too https://v4.mui.com/customization/theming/

**Comment by jangolano** [comment]
We have a case where we would like to control the height of the cards on an individual basis.  We have some custom screens within the application.

**Comment by freben** [comment]
I still wonder if it's possible to achieve the same with for example [theme nesting](https://v4.mui.com/styles/advanced/#theme-nesting) as per the docs, where you override only the card properties part of the theme.

File: packages/core-components/src/layout/TabbedCard/TabbedCard.tsx [hunk 1]
```diff
@@ -61,6 +62,7 @@ const BoldHeader = withStyles(
 type Props = {
   /** @deprecated Use errorBoundaryProps instead */
   slackChannel?: string;
+  style?: CSSProperties;
```
Comment (camilaibs) [inline packages/core-components/src/layout/TabbedCard/TabbedCard.tsx:65]: I would recommend using `className` instead as we could apply to multiple elements conceptually. Is there a particular reason to use style?

**Review by camilaibs — COMMENTED** [review]
Hi, good to see you joining the contributors community.
Left a initial question comment:

**Pull request closed without merging**